regression-test/suites/fault_injection_p0/cloud/test_delete_bitmap_lock_with_restart.groovy

```diff
@@ -0,0 +1,316 @@
+            waitForCompaction(backendId_to_backendIP[trigger_backend_id], backendId_to_backendHttpPort[trigger_backend_id], tablet_id)
+            getTabletStatus(backendId_to_backendIP[trigger_backend_id], backendId_to_backendHttpPort[trigger_backend_id], tablet_id)
+        }
+    }
+    //3. sc
+    docker(options) {
+        def getJobState = {
+            def res = sql_return_maparray "SHOW ALTER TABLE COLUMN WHERE TableName='${tableName}' ORDER BY createtime DESC LIMIT 1"
+            assert res.size() == 1
+            log.info("res:" + res[0].State)
+            return res[0].State
+        }
+        sql """ drop table if exists ${tableName}; """
+
+        sql """
+        CREATE TABLE `${tableName}` (
+            `id` int(11) NOT NULL,
+            `name` varchar(10) NULL,
+            `score` int(11) NULL
+        ) ENGINE=OLAP
+        UNIQUE KEY(`id`)
+        DISTRIBUTED BY HASH(`id`) BUCKETS 1
+        PROPERTIES (
+            "disable_auto_compaction" = "true",
+            "enable_unique_key_merge_on_write" = "true",
+            "replication_num" = "1"
+        );
+        """
+        sql """ INSERT INTO ${tableName} (id, name, score) VALUES (1, "AAA", 15);"""
+        sql """ INSERT INTO ${tableName} (id, name, score) VALUES (2, "BBB", 25);"""
+        sql """ INSERT INTO ${tableName} (id, name, score) VALUES (3, "CCC", 35);"""
+        GetDebugPoint().enableDebugPointForAllBEs("CloudMetaMgr.get_delete_bitmap_update_lock.inject_sleep", [percent: "1.0", sleep: "10"])
+        sql "alter table ${tableName} modify column score varchar(100);"
+        // 1. sc + restart fe
+        cluster.restartFrontends()
+        context.reconnectFe()
+        for (int i = 0; i < 30; i++) {
+            log.info("i: ${i}")
+            try {
+                def now = System.currentTimeMillis()
+                sql """ INSERT INTO ${tableName} (id, name, score) VALUES (1, "AAA", 15);"""
+                def time_cost = System.currentTimeMillis() - now
+                log.info("time_cost(ms): ${time_cost}")
+                assertTrue(time_cost < 10000, "wait time should less than 10s")
+                break
+            } catch (Exception e) {
+                log.info("Exception:" + e)
+                Thread.sleep(2000)
+            }
+        }
+        int max_try_time = 30
+        while (max_try_time--) {
+            def result = getJobState(tableName)
+            if (result == "FINISHED" || result == "CANCELLED") {
+                break
+            } else {
+                Thread.sleep(1000)
+            }
+        }
+        // 2. sc + restart be
+        sql "alter table ${tableName} modify column score varchar(200);"
+        cluster.restartBackends()
+        def now = System.currentTimeMillis()
+        do_stream_load()
+        def time_cost = System.currentTimeMillis() - now
+        log.info("time_cost(ms): ${time_cost}")
+        assertTrue(time_cost > 10000, "wait time should less than 10s")
```

Comment:
![image](https://github.com/user-attachments/assets/51210951-fa60-490f-ac1e-45f0183533fc)
`do_stream_load` asserts that the duration is less than 10s. 
Here, the test case asserts that the `time_cost` is bigger than 10s. 
Can this test case pass?

Reply:
my test show it can pass
2025-04-07 17:52:30.290 INFO [docker-suite-thread-1] (StreamLoadAction.groovy:201) - Stream load elapsed 44377 ms, is http stream: false,  response: {
    "TxnId": 69848588288,
    "Label": "8f4f7f1d-ac02-42ec-ba11-eaed2c2c5971",
    "Comment": "",
    "TwoPhaseCommit": "false",
    "Status": "Success",
    "Message": "OK",
    "NumberTotalRows": 2,
    "NumberLoadedRows": 2,
    "NumberFilteredRows": 0,
    "NumberUnselectedRows": 0,
    "LoadBytes": 15,
    "LoadTimeMs": 44115,
    "BeginTxnTimeMs": 88,
    "StreamLoadPutTimeMs": 292,
    "ReadDataTimeMs": 0,
    "WriteDataTimeMs": 135,
    "ReceiveDataTimeMs": 382,
    "CommitAndPublishTimeMs": 43599
}